package.json

```diff
@@ -117,7 +117,6 @@
         "babel-jest": "29.4.3",
         "cross-env": "^7.0.3",
         "css-loader": "^6.4.0",
-        "cypress": "^12.2.0",
```

Comment:
i think this one gets added automatically by Nx every time you generate a new next or react lib..